src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1191 @@
+                { InternationalFixedDate.of (2012, true, false), ChronoField.DAY_OF_YEAR,         169, 169 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.DAY_OF_YEAR,         366, 366 },
+
+                { InternationalFixedDate.of (2011, false, true), ChronoField.MONTH_OF_YEAR,         0,   0 },
+                { InternationalFixedDate.of (2012, true, false), ChronoField.MONTH_OF_YEAR,         0,   0 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.MONTH_OF_YEAR,         0,   0 },
+
+                { InternationalFixedDate.of (2011, false, true), ChronoField.ALIGNED_WEEK_OF_MONTH, 0,  0 },
+                { InternationalFixedDate.of (2012, true, false), ChronoField.ALIGNED_WEEK_OF_MONTH, 0,  0 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.ALIGNED_WEEK_OF_MONTH, 0,  0 },
+
+                { InternationalFixedDate.of (2011, false, true), ChronoField.PROLEPTIC_MONTH,      13, 12_999_999 },
+                { InternationalFixedDate.of (2012, true, false), ChronoField.PROLEPTIC_MONTH,      13, 12_999_999 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.PROLEPTIC_MONTH,      13, 12_999_999 },
+
+                { InternationalFixedDate.of (2011, false, true), ChronoField.YEAR_OF_ERA,           1,    999_999 },
+                { InternationalFixedDate.of (2012, true, false), ChronoField.YEAR_OF_ERA,           1,    999_999 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.YEAR_OF_ERA,           1,    999_999 },
+
+                { InternationalFixedDate.of (2011, false, true), ChronoField.YEAR,                  1,    999_999 },
+                { InternationalFixedDate.of (2012, true, false), ChronoField.YEAR,                  1,    999_999 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.YEAR,                  1,    999_999 },
+
+                { InternationalFixedDate.of (2011, false, true), ChronoField.EPOCH_DAY,      -719_528, 365_000_000 },
+                { InternationalFixedDate.of (2012, true, false), ChronoField.EPOCH_DAY,      -719_528, 365_000_000 },
+                { InternationalFixedDate.of (2012, false, true), ChronoField.EPOCH_DAY,      -719_528, 365_000_000 },
+        };
+    }
+
+    @Test(dataProvider = "rangesLeapAndYearDay")
+    public void test_range_leap_and_year_day(
+            final InternationalFixedDate date,
+            final TemporalField field,
+            final int expectedMin,
+            final int expectedMax) {
+        assertEquals(date.range(field), ValueRange.of(expectedMin, expectedMax));
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.getLong
+    //-----------------------------------------------------------------------
+    @DataProvider(name = "getLong")
+    Object[][] data_getLong() {
```

Comment:
Only bother testing leap-day/year-day instances for values that should be different, like `DAY_OF_WEEK` (which is in there twice, for whatever reason).